README.md

```diff
@@ -3,25 +3,49 @@ Azure Functions prepared for the Mapping Viz Tools project. This repo contains [
 
 More info on the Mapping Viz Tools [wiki](https://github.com/zooniverse/mapping-viz-tools/wiki/What-are-Azure-Functions%3F)
 
+## Getting started
 ### Dependencies
 In order to run this code, you'll need to install the Azure Functions [core tools.](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local?tabs=macos%2Ccsharp%2Cbash#v2)
 
 Although not required, you may also want to install the Azure VS Code [extension.](https://code.visualstudio.com/docs/azure/extensions)
 
-### yarn install
+### Local database setup
+1. Install PostgreSQL 9.5 and run the PostgreSQL service locally 
+2. Create a database called `mapping_viz_local`. You can do so with the following command on the terminal:  
+`createdb mapping_viz_local`
+3. Once that database is created, execute the db/schema.sql file and db/test_data.sql file on the `mapping_viz_local` database. To do so, first connect to the `mapping_viz_local` database via `psql` on the command line:  
+`psql -d mapping_viz_local`  
```

Comment:
this can collapse into one statement
```
psql -d mapping_viz_local < ./db/schema.sql
```
If using docker-compose, mount the db volume and then run the cmd
```
docker-compose run -v ./db:/db --rm postgres psql -d mapping_viz_local < /db/schema.sql
```